src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1739,6 +1742,13 @@ def create_vmss(vmss_name, resource_group_name, image,
     if secrets:
         secrets = _merge_secrets([load_json(secret) for secret in secrets])
 
+    if enable_autoscale:
```

Comment:
Can someone `disable autoscale-setting` on the resource after enabling it from this argument?